app/src/main/res/values/colors.xml

```diff
@@ -19,5 +18,7 @@
     <color name="darkBurgundyHighlights">#990000</color>
     <color name="lightGray">#d1d3d3</color>
     <color name="mediumGray">#8e979d</color>
+    <color name="shadeOne">#CDB6A3</color>
+    <color name="shadeTwo">#D7C5B5</color>
```

Comment:
What are these shades used for - Not descriptive and does not clarify where they will be used. Others who might want to use a consistent theme might not know which colour to pick.

Reply:
They are for setting the background color of the event text views. They alternate between days. I will rename them.